cypress-tests/cypress/e2e/PaymentTest/00020-Variations.cy.js

```diff
@@ -0,0 +1,275 @@
+import * as fixtures from "../../fixtures/imports";
+import State from "../../utils/State";
+import getConnectorDetails from "../PaymentUtils/Utils";
+
+let globalState;
+let paymentIntentBody;
+let paymentCreateConfirmBody;
+
+describe("Corner cases", () => {
+  // This is needed to get flush out old data
+  beforeEach("seed global state", () => {
+    paymentIntentBody = Cypress._.cloneDeep(fixtures.createPaymentBody);
+    paymentCreateConfirmBody = Cypress._.cloneDeep(
+      fixtures.createConfirmPaymentBody
+    );
+  });
+
+  context("[Payment] [Payment create] Invalid Card Info", () => {
+    before("seed global state", () => {
+      cy.task("getGlobalState").then((state) => {
+        globalState = new State(state);
+      });
+    });
+
+    after("flush global state", () => {
+      cy.task("setGlobalState", globalState.data);
+    });
+
+    it("[Payment create] Invalid card number", () => {
+      let data = getConnectorDetails(globalState.get("commons"))["card_pm"][
```

Comment:
Using `commons` in here is intentional since we expect these to run against regardless of connector